src/server/pfs/server/driver.go

```diff
@@ -531,7 +532,8 @@ func (d *driver) makeCommit(pachClient *client.APIClient, ID string, parent *pfs
 		}
 
 		// BuildCommit case: Now that 'parent' is resolved, read the parent commit's
-		// tree (inside txn) and update the repo size
+		// tree (inside txn)
```

Comment:
I think this comment should be below the line below, or maybe just inside the if block since I think that's what it applies to and it would prevent it getting separated from the code it's commenting on.